### DISCUSSION TITLE
Apple LED Cinema Display Auto Brightness

### DISCUSSION THREAD

**Discussion by ejb068** [MonitorControl/MonitorControl#967] [Q&A] [answered]
Hope someone can help.

I have a 27" LED Cinema display connects to a 2022 MacBook Pro 14" via a usb-c to mini display port adapter. The computer is running 12.2.1.

The MonitorControl app has been great as now I can control brightness (at least partially). I originally thought my display was getting old because it isn't as bright as my slightly newer 27" thunderbolt display. However, I just installed a powerful work light on my office and when I turn it on, I see the brightness slowly increase on the LED Cinema Display. When I turn off the light, the display slowly dims down again. All the time the brightness slider is at max in the MontorControl app. Looks like auto-brightness is enabled. How do I disable it? Or even better get full range of brightness even in a dim room?

I did notice if I play with the brightness setting while the light is on, the display dims (~50%) even when set to max. I have to turn the light off then back on again for the brightness to increase to full again.

Obviously the display is cable of higher brightness as it does this automatically in bright light. How do I get control of it?

Thanks in advance for your help!

-James

**Comment by waydabber** [comment, marked as answer]
Hi @ejb068, see these: https://github.com/MonitorControl/MonitorControl/discussions/891, https://github.com/MonitorControl/MonitorControl/discussions/896 and other discussions related to this. For some specifying a max ddc value of 255 helps but for some doesn't, it's probably dependent on hardware and/or firmware revision of the display. If brightness is changing for 0-50%, then you might be experiencing only software dimming, not actual hardware dimming.

Hope this helps!

**Comment by ejb068** [comment]
Thank you. Setting to 255 worked!

Add Support for old Apple Cinema Displays (USB Brightness Control) https://github.com/MonitorControl/MonitorControl/discussions/891,
